fdbserver/masterserver.actor.cpp

```diff
@@ -1251,6 +1260,14 @@ ACTOR Future<Void> serveLiveCommittedVersion(Reference<MasterData> self) {
 				++self->reportLiveCommittedVersionRequests;
 				req.reply.send(Void());
 			}
+			when(GetTLogPrevCommitVersionRequest req =
+			         waitNext(self->myInterface.getTLogPrevCommitVersion.getFuture())) {
+				GetTLogPrevCommitVersionReply reply;
```

Comment:
Maybe add a TODO here, as the reply needs to be ordered just like `waitForPrev`.